modules/power-monitoring-metrics-overview.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/visualizing-power-monitoring-metrics.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="power-monitoring-metrics-overview_{context}"]
+= {PM-shortname-c} metricss overview
+
+The {PM-operator} exposes the following metrics, which you can view by using the {OCP} web console under the *Observe* -> *Metrics* tab.
+
+[WARNING]
+====
+This list of exposed metrics is not definite. Some metrics can be added or removed in the future releases.
```

Comment:
This might be the case of me not being native English speaker, but I am trying to say that the list might change in the future as some metrics can be added or removed, and now I am not sure how it sounds. As you are away, I will discuss this with a native English speaker to make sure it said what it should, thank you!

Reply:
Hello @briandooley can you help me with this as @bergerhoffer in on PTO until Thursday?

Reply:
I suggest [definitive](https://www.merriam-webster.com/dictionary/definitive) instead of definite.

Reply:
Thank you!